priv/repo/migrations/20181022100328_associate_manuals_to_bookmark.exs

```diff
@@ -0,0 +1,11 @@
+defmodule QbBackend.Repo.Migrations.AssociateManualsToBookmark do
+  use Ecto.Migration
+
+  def change do
+    alter table(:manuals) do
+      add :bookmark_id, references(:bookmarks, type: :binary_id)
+    end
+
+    create index(:manuals, [:bookmark_id])
```

Comment:
@WanjikuMac  we likely don't need this index any more since I think this is in the join table

Reply:
Yea the tests are passing without it.